pkg/config/config.go

```diff
@@ -352,6 +352,8 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("logs_config.open_files_limit", 100)
 	// add global processing rules that are applied on all logs
 	config.BindEnv("logs_config.processing_rules")
+	// enforce the agent to use files to collect container logs on kubernetes environment
+	config.BindEnvAndSetDefault("logs_config.k8s_container_use_file", false)
```

Comment:
Is it the correct milestone? Looks like added feature 🤔

Reply:
this is a fix to a problem that has been introduced to `6.11.2` and which will arise much more in `6.12` if we don't do anything.